### ISSUE TITLE
执行umi setup  一直挂着

### ISSUE THREAD

**Issue by waltcow** [umijs/umi-next#616]
<img width="1419" alt="image" src="https://user-images.githubusercontent.com/3941174/163532879-de1dfdf1-ff13-4984-a273-74f26cf17b0a.png">

**Comment by waltcow** [comment]
配置中使用了tailwind的插件
```js
{
  npmClient: 'pnpm',

  plugins: [require.resolve('@umijs/plugins/dist/tailwindcss')],

  tailwindcss: {},

  fastRefresh: true
}
```

注释`tailwindcss`后正常退出

**Comment by waltcow** [comment]
fixed with "@umijs/plugins" update to   "4.0.0-rc.12"
